Update vpn-gateway-troubleshoot-site-to-site-cannot-connect.md

ShuheiUda opened the pull request:
encoded url is not user friendly.
(https://%3Cyourvirtualnetworkgatewayip%3E:8081/healthprobe)

azurecla commented:
Hi __@LuisCabrer__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (luisca). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

ShuheiUda committed:
Update vpn-gateway-troubleshoot-site-to-site-cannot-connect.md

encoded url is not user friendly.
(https://%3Cyourvirtualnetworkgatewayip%3E:8081/healthprobe)
